src/test/java/org/mule/extension/db/integration/model/OracleTestDatabase.java

```diff
@@ -430,4 +430,26 @@ protected String getInsertRegionManagerSql(RegionManager regionManager) {
 
     return builder.toString();
   }
+
+  public void createPersonTable(Connection connection) throws SQLException {
+    createPersonType(connection);
+    executeDdl(connection, "create or replace type PERSON_TABLE as table of PERSON_TYPE;");
+  }
+
+  public void createPersonType(Connection connection) throws SQLException {
+    executeDdl(connection, "create or replace type PERSON_TYPE as object ( " +
+        "personId varchar2(36), " +
+        "name clob, " +
```

Comment:
Though this does not affect the test, it may seem strange to use a CLOB for a name field.